### DISCUSSION TITLE
Improvement of Report template

### DISCUSSION THREAD

**Discussion by sohamakl** [typemill/typemill#475] [Ideas]
Could you add an option in the report template configuration to ensure that the last page is always an even page, i.e. insert a blank page before the last page if necessary?

**Comment by sohamakl** [comment]
What do you think about the request.

**Reply by trendschau** [reply]
To be honest, I don't understand this requirement, because you can already add or remove empty pages to main chapters and content elements like toc, which is common in print-book publications. Adding it just to one element sounds like a very individual request that should not go into a layout for general use. So in my eyes the right way here is to copy the layout and add this feature to an individual layout.
